### PR TITLE
Update package H5Z-ZFP to 1.1.1

### PR DESCRIPTION
**Pull request by markcmiller86**

### CODE REVIEW

File: var/spack/repos/builtin/packages/h5z-zfp/package.py [hunk 1]
```diff
@@ -66,3 +70,8 @@ def build_targets(self):
     def install_targets(self):
         make_args = ["install"]
         return make_args + self.make_defs
+
+    @run_after("build")
```
Comment (balay) [inline var/spack/repos/builtin/packages/h5z-zfp/package.py:74]: I'm guessing this should be after("install") - will defer it to experts @adamjstewart to confirm..

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/h5z-zfp/package.py:74]: `make check` usually comes after `make` and before `make install` so this looks right

**Comment by balay** [comment]
@spackbot run pipeline

**Review by balay — APPROVED** [review]

**Review by alecbcs — APPROVED** [review]
Confirmed SHAs. Looks good to me.

**Pull request merged**